connectors/src/api/slack_channels_linked_with_agent.ts

```diff
@@ -51,9 +52,12 @@ const _patchSlackChannelsLinkedWithAgentHandler = async (
   const {
     connector_id: connectorId,
     agent_configuration_id: agentConfigurationId,
-    slack_channel_ids: slackChannelIds,
+    slack_channel_ids: slackChannelInternalIds,
```

Comment:
Ideally we would rename this parameter?

Reply:
Basically the parameter should be named slack_channel_internal_ids I presume?

Reply:
✅